lib/kdump_utils.pm

```diff
@@ -164,4 +170,130 @@ sub do_kdump {
     script_run "echo c > /proc/sysrq-trigger", 0;
 }
 
+sub install_service {
+    zypper_call("in kdump");
+}
+
+# for kdump service we need use yast2 kdump to enable it
+# and reboot to make kdump service enabled and activated.
+sub enable_service {
+    activate_kdump();
+}
+
+sub start_service {
+    systemctl 'start kdump';
+}
+
+# check service is running and enabled
+sub check_service {
+    systemctl 'is-enabled kdump.service';
+    systemctl 'is-active kdump';
```

Comment:
function `kdump_is_active` already exists and does much better job

Reply:
we migration need check the kdump service is enabled and activated after started and after migration. we do not need to restart it. this is just for our migration use.

Reply:
It is not enough for kdump to check its status by this way and it is duplication, because when `kdump_is_active` function passed it is clear that service was enabled and running.

Reply:
OK, we'll use kdump_is_active to check the service.